product_secondary_unit/tests/test_product_second_unit.py

```diff
@@ -0,0 +1,158 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import logging
+
+from odoo.fields import Command
+from odoo.tests import TransactionCase, tagged
+
+_logger = logging.getLogger(__name__)
+
+
+@tagged("post_install", "-at_install")
+class TestProductSecondaryUnit(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.product_uom_kg = cls.env.ref("uom.product_uom_kgm")
+        cls.product_uom_unit = cls.env.ref("uom.product_uom_unit")
+        cls.product = cls.env["product.template"].create(
+            {
+                "name": "test",
+                "uom_id": cls.product_uom_kg.id,
+                "uom_po_id": cls.product_uom_kg.id,
+                "secondary_uom_ids": [
+                    (
+                        0,
+                        0,
+                        {
+                            "code": "A",
+                            "name": "unit-700",
+                            "uom_id": cls.product_uom_unit.id,
+                            "factor": 0.7,
+                        },
+                    ),
+                    (
+                        0,
+                        0,
+                        {
+                            "code": "B",
+                            "name": "unit-900",
+                            "uom_id": cls.product_uom_unit.id,
+                            "factor": 0.9,
+                        },
+                    ),
```

Comment:
Could you use the [Command](https://github.com/odoo/odoo/blob/370de95190cf07f17281f0ed6365b6806663cac2/odoo/fields.py#L4227-L4256) class? It is more readable. And check the remaining code, please.
```suggestion
                    Command.create(
                        {
                            "code": "A",
                            "name": "unit-700",
                            "uom_id": cls.product_uom_unit.id,
                            "factor": 0.7,
                        },
                    ),
                   Command.create(
                        {
                            "code": "B",
                            "name": "unit-900",
                            "uom_id": cls.product_uom_unit.id,
                            "factor": 0.9,
                        },
                    ),
```